frontend/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  plugins: ["prettier"],
+  extends: ["react-app", "react-app/jest", "plugin:prettier/recommended"],
```

Comment:
Funktioniert das erweitern der `react-app` config korrect? da z.B. wenn man `useEffect` mit einer Funktion verwendet die nicht in dem Dependecie Array steht keine Warnung bekommt.

Reply:
Ja das funktioniert, "react-app" enthält aber nicht diese Regel. Habe es hinzugefügt :+1: